app/src/App.js

```diff
@@ -12,10 +13,13 @@ const App = () => {
   }, []);
 
   return (
-    <main className="App">
-      <TaskList tasks={tasks} />
-      <AddTask loadTasks={loadTasks} />
-    </main>
+    <Fragment>
```

Comment:
this can also be an empty tag, but fragment works, too!  :)

<Fragment> === <>